packages/catalog-generator/src/main/java/io/kaoto/camelcatalog/Build.java

```diff
@@ -0,0 +1,26 @@
+package io.kaoto.camelcatalog;
+
+import java.util.logging.Logger;
+
+import org.apache.camel.catalog.CamelCatalog;
+import org.apache.camel.catalog.DefaultCamelCatalog;
+
+import io.kaoto.camelcatalog.beans.ConfigBean;
+import io.kaoto.camelcatalog.commands.GenerateCommand;
+import io.kaoto.camelcatalog.commands.GenerateCommandOptions;
+
+public class Build {
+    private static final Logger LOGGER = Logger.getLogger(Build.class.getName());
+
+    public static void main(String[] args) {
+        CamelCatalog catalog = new DefaultCamelCatalog();
+        String camelMainVersion = catalog.getCatalogVersion();
+
+        LOGGER.info("Building Camel Main Catalog: " + camelMainVersion);
```

Comment:
```suggestion
        LOGGER.log(Level.INFO, () -> "Building Camel Main Catalog: {}" + camelMainVersion);
```

Sonar: Use the built-in formatting to construct this argument.